script.js

```diff
@@ -75,18 +87,27 @@ const movements = [200, 450, -400, 3000, -650, -130, 70, 1300];
 
 /////////////////////////////////////////////////
 
-// displays the movements - deposit, withdrawal etc.
+// displays the movements - deposit, withdrawal etc. && the dates
 const displayMovements = function(account, sort = false) {
-    const movs = sort ? account.movements.slice().sort((a, b) => a - b) : account.movements
+    // displays the date under the "Current Balance"
+    const date = new Date()
+    labelDate.textContent = `${date.getMonth().toString().padStart(2, '0')}/${date.getDate().toString().padStart(2, '0')}/${date.getFullYear()}`
```

Comment:
this logic is repeated twice maybe extract to a function and call it in each place?